static/js/settings_invites.js

```diff
@@ -97,8 +97,8 @@ function populate_invites(invites_data) {
 }
 
 function do_revoke_invite() {
-    const modal_invite_id = $(".dialog_submit_button").attr("data-invite-id");
-    const modal_is_multiuse = $(".dialog_submit_button").attr("data-is-multiuse");
+    const modal_invite_id = $("#dialog_submit_button").attr("data-invite-id");
+    const modal_is_multiuse = $("#dialog_submit_button").attr("data-is-multiuse");
```

Comment:
Similarly, if we can migrate from using a class to an ID for these submit buttons in a preparatory commit, that'd be helpful.

That said, doesn't this approach prevent us from rendering a modal on top of another modal?  I wonder if we want to stick with CSS classes rather than IDs here.

Reply:
> That said, doesn't this approach prevent us from rendering a modal on top of another modal? I wonder if we want to stick with CSS classes rather than IDs here.

Modals on modals feel like a bad design nor do we have a use case for it yet :)

Anyway, I went with classes here.

Reply:
We do have a use case for it from a UI goals perspective in a small corner case; so it would help to avoid making it hard to add in the future.